diff --git a/index.js b/index.js
index ab5622e11cd0b86a499c76132aa385c92a8ffedf..64d02d1748beaa7d55fc8250e2357993abcd1ae4 100644
--- a/index.js
+++ b/index.js
@@ -60,7 +60,7 @@ const workerRegexpForOutput = /new\s+Worker\(new\s+URL\((?:'.*?'|".*?"),\s*modul
 
 let longWarningAlreadyShown = false;
 
-module.exports = function(opts = {}) {
+module.exports = function (opts = {}) {
   opts = Object.assign({}, defaultOpts, opts);
 
   opts.loader = ejs.render(opts.loader, opts);
@@ -222,7 +222,8 @@ This will become a hard error in the future.`,
         // In ESM, we never prepend a loader.
         isEsmOutput = () => true;
       } else if (format !== "amd") {
-        this.error(
+        isEsmOutput = () => false;
+        this.warn(
           `\`output.format\` must either be "amd" or "esm", got "${format}"`
         );
       } else {
